### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by anhquoc831997**
image

### CODE REVIEW

**Comment by anhquoc831997** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+gem 'jquery-rails'
+gem 'mini_magick'
+gem 'puma', '~> 3.11'
+gem 'rails', '~> 6.0.0.rc1'
+gem 'rails-controller-testing'
+gem 'rubocop', '~> 0.54.0', require: false
+gem 'sass-rails', '~> 5.0'
+gem 'turbolinks', '~> 5'
+gem 'uglifier', '>= 1.3.0'
+gem 'will_paginate', '3.1.7'
```
Comment (phamvanbk11) [inline Gemfile]: Nháy ""

**Comment by phamvanbk11** [comment]
commented

**Comment by anhquoc831997** [comment]
ready

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'spring'
+  gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'web-console', '>= 3.3.0'
+end
+
+group :test do
+  gem 'capybara', '>= 2.15'
+  gem 'selenium-webdriver'
+  gem 'webdrivers'
```
Comment (phamvanbk11) [inline Gemfile]: Chưa fix nháy " nhé

**Comment by phamvanbk11** [comment]
commented

**Comment by anhquoc831997** [comment]
Da sr anh. Em fix roi ma khong hieu sao van con. Em fix lai roi a.

**Comment by anhquoc831997** [comment]
ready

**Pull request merged**